examples/psql/schema.sql

```diff
@@ -0,0 +1,69 @@
+CREATE TABLE IF NOT EXISTS benchmarks (
+    name TEXT PRIMARY KEY
+);
+
+CREATE TABLE IF NOT EXISTS branches (
+    name TEXT PRIMARY KEY
+);
+
+CREATE TABLE IF NOT EXISTS stores (
+    name TEXT PRIMARY KEY
+);
+
+CREATE TABLE IF NOT EXISTS instance_types (
+    name TEXT PRIMARY KEY
+);
+
+CREATE TABLE IF NOT EXISTS scenarios (
+    name TEXT PRIMARY KEY
+);
+
+CREATE TABLE IF NOT EXISTS configs (
+    id SERIAL PRIMARY KEY,
+    benchmark TEXT NOT NULL REFERENCES benchmarks(name),
+    scenario TEXT NOT NULL REFERENCES scenarios(name),
+    store TEXT NOT NULL REFERENCES stores(name),
+    instance_type TEXT NOT NULL REFERENCES instance_types(name),
+    cache BOOLEAN NOT NULL,
+    UNIQUE(benchmark,
+           scenario,
+           store,
+           cache,
+           instance_type)
+);
+
+CREATE TABLE IF NOT EXISTS experiments (
+    id TEXT PRIMARY KEY,
+    ts TIMESTAMPTZ NOT NULL,
+    branch TEXT NOT NULL REFERENCES branches(name),
+    commit TEXT NOT NULL,
+    commit_ts TIMESTAMPTZ NOT NULL,
+    username TEXT NOT NULL,
+    details_url TEXT NOT NULL,
+    exclude_from_analysis BOOLEAN DEFAULT false NOT NULL,
+    exclude_reason TEXT
+);
+
+CREATE TABLE IF NOT EXISTS results (
+  experiment_id TEXT NOT NULL REFERENCES experiments(id),
+  config_id INTEGER NOT NULL REFERENCES configs(id),
+
+  samples BIGINT NOT NULL,
```

Comment:
Values from here down seems novel / new to me... What did you add or alternatively what did I forget?

Reply:
`samples` contains the number of runs used to produce aggregate statistics. I found it useful to confirm or refute that a possible anomaly is caused by failed runs. 

Since the goal of this example is to show how to make hunter work with PostgreSQL, I removed `samples` in [139c075](https://github.com/Gerrrr/hunter/pull/13/commits/139c0750e9d9955a6571698077562ce414f722b5).